app/serializers/concerns/navigation_menu_tags_mixin.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module NavigationMenuTagsMixin
+  def serialize_tags(tags)
+    topic_count_column = Tag.topic_count_column(scope)
+
+    tags
+      .select(:name, topic_count_column, :pm_topic_count, :description)
+      .order(topic_count_column => :desc)
```

Comment:
```suggestion
      .order(topic_count_column: :desc)
```

Reply:
I don't think is the same. `topic_count_column` is a variable storing a symbol that may not be `:topic_count_column`.